src/registrar/templates/django/admin/domain_request_change_form.html

```diff
@@ -4,7 +4,17 @@
 {% block field_sets %}
     {# Create an invisible <a> tag so that we can use a click event to toggle the modal. #}
     <a id="invisible-ineligible-modal-toggler" class="display-none" href="#toggle-set-ineligible" aria-controls="toggle-set-ineligible" data-open-modal></a>
-    {{ block.super }}
+
+    {% for fieldset in adminform %}
+        {% comment %}
+        TODO: this will eventually need to be changed to something like this
```

Comment:
Is this something we would want to make a ticket for if there are later plans to customize the display of the analyst dashboard? cc @abroddrick